scm-ui/ui-api/src/base.ts

```diff
@@ -31,6 +31,7 @@ import { requiredLink } from "./links";
 
 export type ApiResult<T> = {
   isLoading: boolean;
+  isFetching: boolean;
```

Comment:
This breaks some other typings resulting in a broken build. Shouldn't `isFetching` be optional?

Reply:
At least this breaks the build. Normally this should be taken from the default react-query result, shouldn't it?

Reply:
I would say so. Could you create a new type "ApiResultWithFetching(?)" which extends  ApiResult?